crates/ty_server/src/server/api/requests/goto_definition.rs

```diff
@@ -0,0 +1,73 @@
+use std::borrow::Cow;
+
+use lsp_types::GotoDefinitionParams;
+use lsp_types::request::GotoDefinition;
+use lsp_types::{GotoDefinitionResponse, Url};
+use ruff_db::source::{line_index, source_text};
+use ty_ide::goto_definition;
+use ty_project::ProjectDatabase;
+
+use crate::DocumentSnapshot;
+use crate::document::{PositionExt, ToLink};
+use crate::server::api::traits::{BackgroundDocumentRequestHandler, RequestHandler};
+use crate::session::client::Client;
+
+pub(crate) struct GotoDefinitionRequestHandler;
+
+impl RequestHandler for GotoDefinitionRequestHandler {
+    type RequestType = GotoDefinition;
+}
```

Comment:
Everything in this file is a copy of goto_type_definition but changed to goto_definition